lib/admin/generate_uploads.dart

```diff
@@ -230,7 +230,11 @@ Map<String, List<List<String>>> generateJobplan(JobPlanMaximo jobplan,
       joblabor.hours.toString(),
       joblabor.quantity.toString(),
       orgID,
-    ]);
+    ];
+    if (joblabor.laborCode != '') {
+      jobLaborList.add(joblabor.laborCode);
+    }
+    generated['JobLabor']!.add(jobLaborList);
```

Comment:
it doesnt make sense to switch convention for 1 function
and what is the advantage to using this new convention?